easybuild/easyconfigs/a/AmberTools/AmberTools-20-intel-2020a-Python-3.8.2.eb

```diff
@@ -19,26 +19,37 @@ sources = [{
     'download_filename': local_download_credentials,
     'filename': SOURCE_TAR_BZ2,
 }]
-patches = ['Amber-20_fix-hardcoding.patch']
+patches = [
+    'Amber-20_fix-hardcoding.patch',
+    'AmberTools-20_fix_missing_MPI_LIBRARY_error.patch',
+    'AmberTools-20_cmake-locate-netcdf.patch',
+]
 checksums = [
     'b1e1f8f277c54e88abc9f590e788bbb2f7a49bcff5e8d8a6eacfaf332a4890f9',  # AmberTools-20.tar.bz2
     'a71f1bd70bac8d5e25386d6db90e95874ad300b956ed5b5afbc8848b7c1e00dc',  # Amber-20_fix-hardcoding.patch
+    # AmberTools-20_fix_missing_MPI_LIBRARY_error.patch'
+    '0b89a0624167bc23876bcdefcb1055f591e38e3bd559a71d5749e342bd311acc',
+    '473e07c53b6f641d96d333974a6af2e03413fecef79f879d3fdecf7fecaab4d0',  # AmberTools-20_cmake-locate-netcdf.patch
 ]
 
 builddependencies = [
     ('Bison', '3.5.3'),
     ('pkg-config', '0.29.2'),
+    ('CMake', '3.16.4'),
 ]
 dependencies = [
     ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('matplotlib', '3.2.1', versionsuffix),
     ('netCDF', '4.7.4'),
     ('netCDF-Fortran', '4.5.2'),
+    ('FFTW', '3.3.8'),
 ]
 
 # fix linking to netCDF library: also requires linking to HDF5 & cURL libs, which in turns require others,
 # all of which are indirect dependencies via netCDF
 local_netcdf_libs = "-lnetcdf -lhdf5 -lsz -ldl -liomp5 -lcurl -lssl -lcrypto -lz -lm -lpthread"
-preconfigopts = "sed -i'' 's/-lnetcdf/%s/g' AmberTools/src/cpptraj/configure && " % local_netcdf_libs
+preconfigopts = "sed -i'' 's/-lnetcdf/%s/g' %%(builddir)s/AmberTools/src/cpptraj/configure && " % local_netcdf_libs
 configopts = "-nosanderapi"
```

Comment:
What was this for and what is the reason behind adding it in the first place?

Remember AmberTools can be built exactly like Amber, it will just skip the Amber part when it is not available.
So see Amber-20.11-foss-2020a-AmberTools-20.15-Python-3.8.2.eb for instance.

Reply:
Good question, I'm not sure why `AmberTools` was being configured without the `sander` part (I have no clue what it is, tbh)... Seems like it was there from the start, so no trace in Git log.